.golangci.yml

```diff
@@ -1,18 +1,6 @@
 # https://golangci-lint.run/usage/quick-start/
 run:
   timeout: 12m
-  skip-dirs:
-    - dist
-    - docs
-    - examples
-    - hack
-    - manifests
-    - pkg/client
-    - sdks
-    - ui
-    - vendor
-  skip-files:
-    - server/static/files.go
```

Comment:
let's not re-order these in the config, just pure rename them. I'll make that change myself as I'd like to release 3.5.8 today

Reply:
Oh this is in a different section now, moved from `run` to `issues`. Gotcha. Nevermind then, thanks for updating that!